superset/db_engine_specs/bigquery.py

```diff
@@ -141,3 +141,27 @@ def epoch_to_dttm(cls):
     @classmethod
     def epoch_ms_to_dttm(cls):
         return 'TIMESTAMP_MILLIS({col})'
+
+    @classmethod
+    def df_to_sql(cls, df: pd.DataFrame, **kwargs):
+        """
+        Upload data from a Pandas DataFrame to BigQuery. Calls
+        `DataFrame.to_gbq()` which requires `pandas_gbq` to be installed.
+
+        :param df: Dataframe with data to be uploaded
+        :param kwargs: kwargs to be passed to to_gbq() method. Requires both `schema
+        and ``name` to be present in kwargs, which are combined and passed to
+        `to_gbq()` as `destination_table`.
+        """
+        if not ('name' in kwargs and 'schema' in kwargs):
+            raise Exception('name and schema need to be defined in kwargs')
+        gbq_kwargs = {}
+        gbq_kwargs['project_id'] = kwargs['con'].engine.url.host
+        gbq_kwargs['destination_table'] = f"{kwargs.pop('schema')}.{kwargs.pop('name')}"
+
+        # Only pass through supported kwargs
+        supported_kwarg_keys = {'if_exists'}
+        for key in supported_kwarg_keys:
+            if key in kwargs:
+                gbq_kwargs[key] = kwargs[key]
+        df.to_gbq(**gbq_kwargs)
```

Comment:
I'm wondering what the error raised will be if `pandas_gbq` isn't installed. I'm guessing it may be gibberish to most, as in `df object has no attribute to_gbq` which would be confusing to most users and admins.

If that's the case, how about we assert that the lib is installed and if not raise an `ImportError` with a comprehensive message, as in: "Could not import the library `pandas_gbq`, which is required to be installed in your environment in order to load CSV files to BigQuery".

Reply:
It now raises the following error if `pandas_gbq` is missing:
![image](https://user-images.githubusercontent.com/33317356/59974189-c77b3000-95b1-11e9-8531-fc2c3991f22a.png)